ding2.profile

```diff
@@ -760,27 +760,62 @@ function ding2_set_page_not_found() {
  * Add page with std. cookie information.
  */
 function ding2_set_cookie_page() {
+
+  $eu_cookie_compliance_da['popup_enabled'] = TRUE;
+  $eu_cookie_compliance_da['popup_clicking_confirmation'] = FALSE;
+  $eu_cookie_compliance_da['popup_info']['value'] = '<p>Vi bruger cookies på hjemmesiden for at forbedre din oplevelse.</p><p>Læs mere her: <a href="' . url('cookies') . '">Hvad er cookies?</a></p>';
```

Comment:
This relies on the assumption that the path to the page will be `cookies` but I cannot spot anything in the setup of the corresponding node that ensures that.

One way to go about this would be to flip variable setting and node creation and set the link to point to the path for `node/[nid]`.